oap-server-bom/pom.xml

```diff
@@ -39,7 +39,7 @@
         <h2.version>2.1.212</h2.version>
         <joda-time.version>2.10.5</joda-time.version>
         <zookeeper.version>3.5.7</zookeeper.version>
-        <guava.version>31.1-jre</guava.version>
+        <guava.version>32.0.0-jre</guava.version>
```

Comment:
[32.0.1-jre](https://mvnrepository.com/artifact/com.google.guava/guava/32.0.1-jre)

Reply:
Any feature/bug-fix required? This PR is just for a CVE patch only.

Reply:
> Any feature/bug-fix required? This PR is just for a CVE patch only.

io: Fixed Files.createTempDir and FileBackedOutputStream under Windows, which broke as part of the security fix in release 32.0.0. Sorry for the trouble. (https://github.com/google/guava/commit/fdbf77d3f2b826fc0a70b1f9b9994b140ddf3bd8)

Reply:
Done.